wolox-training/nh-android
# Login backend

## Login Backend

### Summary

Implemented service to verify user's credentials using Parse's Rest API.
User inputs email and password, if the values are valid, when the login button is pressed a message is shown telling the user if the login has been successful or not.

**Cards:** 
* Card: [15] https://trello.com/c/Uojf2N7u/15-tech-login

### Test Plan
1) Log in using email "test@mail.com" and password "password", message should be "Login has been successful", as shown in Screenshot 1.
2) Log in using email "test@mail.com" and password "eeeeeeee", message should be "Login error: user or password is incorrect", as shown in Screenshot 2.
3) Log in using email "a@mail.com" and password "password", message should be "Login error: user or password is incorrect", as shown in Screenshot 2.

### Screenshots

#### Screenshot 1
![login-ok](https://cloud.githubusercontent.com/assets/14595211/10195811/e708700a-6765-11e5-8881-d023e4ff0cb5.png)

#### Screenshot 2
![login-error](https://cloud.githubusercontent.com/assets/14595211/10195812/e8388c6c-6765-11e5-9e87-98f306f10f84.png)
